### PR TITLE
Add code of conduct

### PR DESCRIPTION
**Pull request by starbelly**
This PR adds a standard code of conduct (`Contributor Covenant Code of Conduct`) to the repository.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or
+ advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+ address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
```
Comment (ahw59) [inline CODE_OF_CONDUCT.md:31]: Assert comparative or superlative differences of one product, project, company or individual over another

Comment (starbelly) [reply CODE_OF_CONDUCT.md:31]: Yup, can add.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at eef@erlef.org. All
```
Comment (ferd) [inline CODE_OF_CONDUCT.md:60]: gotta make sure this email address routes to something that makes sense.

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+complaints will be reviewed and investigated and will result in a response that
+is deemed necessary and appropriate to the circumstances. The project team is
+obligated to maintain confidentiality with regard to the reporter of an incident.
+Further details of specific enforcement policies may be posted separately.
+
+Project maintainers who do not follow or enforce the Code of Conduct in good
+faith may face temporary or permanent repercussions as determined by other
+members of the project's leadership.
```
Comment (ferd) [inline CODE_OF_CONDUCT.md:68]: should we s/project/foundation/ or this CoC is strictly for the website and is disjoint from the foundation otherwise?

**Review by ferd — APPROVED** [review]

File: CODE_OF_CONDUCT.md (hunk 3 above)
Comment (starbelly) [reply CODE_OF_CONDUCT.md:68]: This CoC is strictly for the website and this repository, at least for now...

**Pull request merged**